frontend/package-lock.json

```diff
@@ -25875,9 +26252,7 @@
         "popper.js": "^1.16.0",
         "prop-types": "15.7.2",
         "raf": "3.4.1",
-        "react": "17.0.1",
```

Comment:
to verify, same with react-dom below because we are using them

Reply:
I don't understand why it removes it here, react and react-dom appear on the dependencies above in lines 19 and 21. These removals are in the *invariant" object, I don't know what it means here,